### PR TITLE
thin_client unit test

### PR DESCRIPTION
**Pull request by bloodelves88**

### CODE REVIEW

File: thin_client_test.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import socket
+from wheel.signatures import assertTrue
+            
+class thin_client_test(unittest.TestCase):
+    # Check if packAndSend increments the sequence number correctly       
+    def test_packAndSend_sequence(self):
+        sequence = 0
+        socketName = socket.socket(socket.AF_INET, # Internet
+                                   socket.SOCK_DGRAM) # UDP
+        finalSequence = thin_client.packAndSend(1, sequence, 0, 0, 0, 0, socketName)
```
Comment (9gix) [inline thin_client_test.py]: in test you may want to use Arrange Act Assert pattern. whereby each test, does this.

**Comment by bloodelves88** [comment]
Used the pattern. Merge if ok.

File: thin_client.py [hunk 2]
```diff
@@ -153,6 +153,8 @@ def initializePygame(FPS):
     screen.blit(mouseLabel, (renderPosX - 100, renderPosY + 50))
     pygame.display.update()
     
+    return True
```
Comment (9gix) [inline thin_client.py]: you shouldn't return `True` just for the sake of return value without any meaning. I believed you need it for testing purposes. 

To solve this in the unit test, you can simply run the function, and handle the error by calling [`fail(msg)`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.fail) method. See example in my comment below.

File: thin_client_test.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        self.assertEqual(finalSequence, 3, "packAndSend sequence from 2 to 3 fail")
+        
+        sequence = 3
+        finalSequence = thin_client.packAndSend(1, sequence, 0, 0, 0, 0, socketName)
+        self.assertEqual(finalSequence, 4, "packAndSend sequence from 3 to 4 fail")
+        
+    # Check if pygame is initialized successfully
+    def test_initializePygame_30FPS(self):
+        assertTrue(thin_client.initializePygame(30), "initializePygame at 30 FPS fail")
```
Comment (9gix) [inline thin_client_test.py]: def test_30_fps(self):
        try:
            thin_client.initializePygame(30)
        except:
            self.fail("Fail to initialize 30 FPS")

    def test_60_fps(self):
        try:
            thin_client.initializePygame(60)
        except:
            self.fail("Fail to initialize 60 FPS")

**Pull request merged**